src/app/Components/LanguageAvatar.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+  export let name: string;
+  export let selectedLanguage: string = "";
+
+  export let isSelected: boolean = name === selectedLanguage;
```

Comment:
**[Question]**: this line of code seems to be very confusing?

Reply:
Thanks for mentioning this. In the `Sidebar` component I loop through each language and show an avatar.

```javascript
  {#each languages as language}
    <LanguageAvatar name={language.name} {selectedLanguage} />
  {/each}
```

 This basically determines if the `selectedLanguage` is the same as the `name` of the language in that avatar so that it can be styled accordingly.